src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftWriter.scala

```diff
@@ -352,6 +352,7 @@ private[redshift] class RedshiftWriter(
       }
     )
 
+    log.info("Unloading data to S3: {}", tempDir)
```

Comment:
@zaza 
Since on line 395 we sanitize the s3 folder in case it has any credential information, can you please remove the logging of the `tempDir` path from the log statement?

Reply:
Done.